### PR TITLE
Broken link

### LINKED ISSUE #88: Broken link on Install MAAS Cli to hardware config info
On the Install MAAS by Packages page there is a link (http://maas.io/docs/orientation#a-typical-maas-setup) to see hardware configuration info and it goes to 404: page not found.

### PR DESCRIPTION
**Pull request by pmatulis**
Fixed link but found a lot of other stuff that needed attention.

Fixes #88

### CODE REVIEW

**Comment by pmatulis** [comment]
Graham, I added the stuff from #80 which I missed somehow and only found it by accident after sending in the current PR. Anyway, I question your recommendation of the ISO install for new users. The ISO will only configure the server as a rack controller or as a region controller. This presumes the user knows what they want and that they are integrating a MAAS piece into an existing MAAS, or at least a well planned out, environment. This doesn't sound like a "new user" to me. IMO the `maas` metapackage is what we want to give to new users. But let's discuss.

**Comment by pmatulis** [comment]
I also removed the repetition of install methods and notes from about-maas. We don't really need it there as well. Thoughts?

File: src/en/installconfig-package-install.md [hunk 1]
```diff
@@ -1,93 +1,73 @@
...
--   [ppa:maas/stable](https://launchpad.net/~maas/+archive/ubuntu/stable)
+## Package repositories
 
-!!! Note: The MAAS team also releases the latest development release of MAAS.
-The development release is available in
-[ppa:maas/next](https://launchpad.net/~maas/+archive/ubuntu/next). 
-However, this is meant to be used for testing and at your own risk.
+While MAAS is available in the normal Ubuntu archives, the available packages
+may be lagging behind non-archive, but still stable, versions. To install a newer
+stable version the follosing PPA can be used:
```
Comment (degville) [inline src/en/installconfig-package-install.md]: typo: follosing/following

**Comment by degville** [comment]
Thanks for all this Peter. Looks good. There's just a tiny typo as commented inline.

I was reluctant to add the repetition of install methods, but I think the slightly expanded descriptions were more appropriate on what is now the landing page (intro-about-maas), rather than the root of the the 'Installation' sub-menus, which I doubt will be accessed much. That was why I thought the repetition was acceptable.

**Comment by pmatulis** [comment]
@degville I added an abridged version of the install methods on about-maas and included a link to the install page. I also did some minor work on the wording of the install methods.

**Comment by degville** [comment]
Excellent solution. Thanks Peter!

**Pull request merged**